app/Http/Controllers/API/CustomTourController.php

```diff
@@ -28,4 +41,20 @@ public function show(Request $request, $id)
         $tour_json = json_decode($custom_tour->tour_json, true);
         return response()->json(['tour_json' => $tour_json], 200);
     }
+
+    private function sanitizeData($data)
```

Comment:
Please add a doc comment

Reply:
I've also added a doc comment explaining what this method does